lib/pr_checker.js

```diff
@@ -175,9 +175,32 @@ class PRChecker {
       logger.warn('No full CI runs detected');
     }
 
+    let lastCI;
     for (const [type, ci] of ciMap) {
       const name = CI_TYPES.get(type).name;
       logger.info(`Last ${name} CI on ${ci.date}: ${ci.link}`);
+      if (!lastCI || lastCI.date > ci.date) {
+        lastCI = {
+          typeName: name,
+          date: ci.date
+        };
+      }
+    }
+
+    if (lastCI) {
+      let didLogHeader = false;
+      commits.forEach((c) => {
+        const commit = c.commit;
+        if (commit.committedDate > lastCI.date) {
+          if (!didLogHeader) {
+            status = false;
+            didLogHeader = true;
+            logger.warn(`Commits pushed after the last ${lastCI.typeName} ` +
+            'CI run:');
+          }
+          logger.warn(`- ${commit.message}`);
```

Comment:
@buji1993 if you log `commit.message` it would log body of message to.
Nit: `commit.message.split('\n')[0]`

Reply:
ok. thanks for your suggestion